resources/twitter_user_scrape.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+import json
+import sys
+from time import sleep
+
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+from selenium.common.exceptions import StaleElementReferenceException
+
+
+class TwitterScrape:
+    TWITTER_IDS_FILES_SUBFIX = "_ids.json"
+    ID_SELECTOR = '.time a.tweet-timestamp'
+    TWEET_SELECTOR = 'li.js-stream-item'
+    TWEETS_PER_PAGE = 10
+    DEFAULT_DELAY_IN_SECONDS = 1
+
+    def twitter_ids_filename(self, user: str):
+        return user + self.TWITTER_IDS_FILES_SUBFIX
+
+    def format_day(self, date):
+        day = '0' + str(date.day) if len(str(date.day)) == 1 else str(date.day)
```

Comment:
Good use of the ternary operator. Albeit I often don't use it because of the readability, it's well employed here.